netlify/vale/CockroachDB/Terminology.yml

```diff
@@ -3,6 +3,8 @@ message: Prefer '%s' over '%s'
 ignorecase: false
 level: warning
 swap:
+  builtin: built-in
+  builtins: built-ins
```

Comment:
lines 6-7 here MVP for this PR >_<

Reply:
I agree. These two errors were the most spammy.